package.json

```diff
@@ -108,5 +108,7 @@
     "webpack-dev-server": "3.1.4",
     "webpack-merge": "4.1.2"
   },
-  "dependencies": {}
+  "dependencies": {
+    "@babel/polyfill": "7.0.0-beta.55"
```

Comment:
should be in dev dependecies ...

Reply:
@sbrunner According to the babel-polyfill doc, no:
> Because this is a polyfill (which will run before your source code), we need it to be a dependency, not a devDependency
But honestly, I don't really know what that mean in our context and in the final code.

Reply:
For us it mens nothing except adding a dependencies will break c2cgeoportal...